src/pymor/models/interface.py

```diff
@@ -179,9 +255,9 @@ def _compute_output_error_estimate(self, solution, mu=None, **kwargs):
 
     _compute_allowed_kwargs = frozenset()
 
-    def compute(self, solution=False, output=False,
-                solution_error_estimate=False, output_error_estimate=False, *,
-                mu=None, **kwargs):
+    def compute(self, solution=False, output=False, solution_d_mu=False, output_d_mu=False,
```

Comment:
Shouldn't the new args come after the "old" ones for compat with positional usage?

Reply:
It is true that is does not preserve positional usage but at the same time I found the order of the arguments way better than just appending them to the current state. IMO, the arguments for the estimates should come after the others...
The `compute` method has anyway also been introduced in the upcoming release which made me think it is okay to still restructure arguments a bit.

Reply:
The recent introduction of the compute method is indeed a very good argument. Aside from that, I have no strong opinion wrt estimates before others or not. 
That also somewhat means we really should get this PR into 2020.2 though.

Reply:
I agree. I also do not really have the strongest opinion about the order of the arguments.

@sdrave , do you have any particular preference?